grammar.js

```diff
@@ -65,15 +65,17 @@ module.exports = grammar({
       "'",
       // NOT optionlink: 'x
       seq("'", token.immediate(/[^'\n\t ]/)),
-      // NOT optionlink: followed by non-lowercase char.
+      // NOT optionlink: 'X (non-lowercase char).
       seq("'", token.immediate(/[a-z]*[^'a-z\n\t ][a-z]*/), optional(token.immediate("'"))),
-      // NOT optionlink: single char surrounded by "'".
+      // NOT optionlink: 'x' (single char).
       seq("'", token.immediate(/[^'\n\t ]/), token.immediate("'")),
       // NOT taglink: "||", "|"
       /\|\|+/,
       '|',
       // NOT listitem: "-" or "•" followed by tab.
       /[-•]\t/,
+      // NOT tag: isolated "*". (Hack to avoid errors.)
+      /\*[\n\t ]/,
```

Comment:
hacky. can be improved later if we figure out a way to make `'*'` work here...
```suggestion
      '*',
```

Reply:
going to try to make this work. because the hacky way leads to other rendering regressions in `gen_help_html.lua`.